app/controllers/api/v2/hosts_controller.rb

```diff
@@ -90,7 +90,7 @@ def show
           param :compute_resource_id, :number, :desc => N_("nil means host is bare metal")
           param :root_pass, String, :desc => N_("required if host is managed and value is not inherited from host group or default password in settings")
           param :model_id, :number
-          param :hostgroup_id, :number
+          param :hostgroup_id, :number, :desc => N_("ID of the host group. Note: Changing the host group will re-apply all attributes inherited from the new host group, unless you explicitly override them again in the update request.")
```

Comment:
Small, but annoying nit:
```suggestion
          param :hostgroup_id, :number, :desc => N_("ID of the host group. Note: Changing the host group will re-apply all attributes inherited from the new host group, unless you explicitly override them again in the update request")
```

Reply:
sure. thanks. :-)